rust-toolchain.toml

```
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.78.0"
+channel = "1.79.0"
```

```suggestion
channel = "1.81.0"
```
https://github.com/anza-xyz/agave/blob/v2.1.13/rust-toolchain.toml